piecrust/src/error.rs

```diff
@@ -60,9 +59,9 @@ pub enum Error {
     MissingHostData(String),
     #[error("Missing host query: {0}")]
     MissingHostQuery(String),
-    #[error("OutOfPoints")]
+    #[error("Contract OutOfPoints")]
```

Comment:
It's not the contract that goes OutOfPoints, it's the execution

Reply:
I was writing a long justification for this, but I realized you're right. However, this also means that the `Error::ContractPanic` should be named `Error::Panic` and the error message adjusted accordingly.